opal/mca/common/ucx/common_ucx.c

```diff
@@ -132,6 +132,28 @@ static void opal_common_ucx_mca_fence_complete_cb(int status, void *fenced)
     *(int*)fenced = 1;
 }
 
+void opal_common_ucx_mca_proc_added(void)
+{
+#if HAVE_DECL_UCM_TEST_EVENTS
+    static int warned = 0;
+    static char *mem_hooks_suggestion = "Try to add command line agrument "
```

Comment:
` Pls try adding  --mca opal_common_ucx_opal_mem_hooks 1 to mpirun/oshrun command line to resolve this issue`

Reply:
message is updated